spec/std/process_spec.cr

```diff
@@ -81,6 +103,22 @@ describe Process do
     $?.exit_code.should eq(0)
   end
 
+  it "chroot raises when unprivileged" do
+    status, output = build_and_run <<-CODE
+      begin
+        Process.chroot("/usr")
+        puts "FAIL"
+      rescue ex : Errno
+        puts ( ex.errno == Errno::EPERM ) ? "Success" : "FAIL: " + (ex.message||"")
```

Comment:
```cr
puts (ex.errno == Errno::EPERM) ? "Success" : "FAIL: #{ex.message}"
```

ditto below

Reply:
There appears to be a bug or something in the language (or I am missing something). It wont allow that. Says "undefined local variable or method 'ex'". Also tried:
```
puts ( ex.errno == Errno::EPERM ) ? "Success" : "FAIL: #{ex.message||"No Message"}"
```
to no avail.